doc/man3/EVP_EncryptInit.pod

```diff
@@ -1338,6 +1338,15 @@ EVP_EncryptUpdate() or EVP_DecryptUpdate() should be made with the output
 parameter I<out> set to B<NULL>. In this case, on success, the parameter
 I<outl> is set to the number of bytes authenticated.
 
+Please note that the number of authenticated bytes returned by
```

Comment:
This is probably at the wrong place in the file. As far as understand, the paragraph above was about adding it when encrypting, while this text seems to be about getting it back when decrypting.

Reply:
Shall I move it below "When decrypting, ..."?

Reply:
I originally placed it there, because the paragraph above describes the behaviour when the parameter `out` is set to NULL.